packages/orchestrator-ui-components/src/utils/getDefaultTableConfig.ts

```diff
@@ -0,0 +1,87 @@
+import {
+    DEFAULT_PAGE_SIZE,
+    ACTIVE_PROCESSES_LIST_TABLE_LOCAL_STORAGE_KEY,
+    COMPLETED_PROCESSES_LIST_TABLE_LOCAL_STORAGE_KEY,
+    METADATA_RESOURCE_TYPES_TABLE_LOCAL_STORAGE_KEY,
+    METADATA_PRODUCTBLOCKS_TABLE_LOCAL_STORAGE_KEY,
+    METADATA_PRODUCT_TABLE_LOCAL_STORAGE_KEY,
+    METADATA_WORKFLOWS_TABLE_LOCAL_STORAGE_KEY,
+    SUBSCRIPTIONS_TABLE_LOCAL_STORAGE_KEY,
+} from '../components';
+
+import type { StoredTableConfig } from '../components';
+
+import {
+    ProductBlockDefinition,
+    ProductDefinition,
+    WorkflowDefinition,
+    Process,
+    SubscriptionListItem,
+} from '../types';
+
+function getTableConfig<T>(
+    hiddenColumns: (keyof T)[] = [],
+    selectedPageSize = DEFAULT_PAGE_SIZE,
+): StoredTableConfig<T> {
+    return {
+        selectedPageSize,
+        hiddenColumns,
+    };
+}
+
+export const getDefaultTableConfig = <T>(storageKey: string) => {
```

Comment:
Looking at the returned data, it seems these are the hidden columns. In that case it might be a good idea to describe that in the function name and in the variable names

Reply:
Ok, but it's not only hidden columns, it's page size to and maybe more to come